src/components/Tabs/Tabs.stories.js

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
+import ClassNames from 'classnames';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text, select, boolean } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import { defaultTemplate } from '../../../storybook/decorators/storyTemplates';
+import {
+  Tabs,
+  Tab,
+  TabContainer,
+  TabPane,
+  TabContent,
+  Col,
+  Row,
+} from '../../index';
+import { Nav, NavItem, NavDropdown, MenuItem } from 'react-bootstrap';
```

Comment:
Can we also export/extend these React Bootstrap components in `src/index.js` to be consistent with other React Bootstrap components used?

Reply:
I know we were just targeting Tabs w/ this PR, but I think it's good to go ahead and export/extend them as the others...